.emacs

```diff
@@ -359,3 +361,17 @@
 (setq-default ediff-highlight-all-diffs 'nil) ;; Only hilight current diff:
 (setq ediff-diff-options "-w")                ;; Turn off whitespace checking:
 (setq ediff-show-clashes-only t)              ;; Default to conflict diff.
+(custom-set-variables
```

Comment:
Please remove those from the commit. Makes the inital package install lost (maybe)